app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="fs-4 p-3 border border-1 container mt-5 p-2 bio">
+  <%= form_with model: @recipe, method: :post, local: true do |f| %>
+    <%= f.label :name, "Name", class: "form-label fs-6" %><br>
+    <%= f.text_field :name, :placeholder => "Add Your Recipe name here...", required: true, class: "form-control" %><br>
+
+    <%= f.label :description, "Write description", class: "form-label fs-6" %>
+    <%= f.text_area :description, size: "80x15", :placeholder => "Add your description here...", required: true, class: "form-control" %>
+
+    <%= f.label :cooking_time, "Cooking time(mins)", class: "form-label fs-6" %>
+    <%= f.text_field :cooking_time, class: "form-control mb-2" %>
+
+    <%= f.label :preparation_time, "Preparation time(mins)", class: "form-label fs-6" %>
+    <%= f.text_field :preparation_time, class: "form-control mb-2" %>
+
+    <div class="text-center">
+    <%= f.label :public, "Make it Public?", class: "form-label fs-6" %>
+    <%= f.check_box :public, class: "mb-2" %>
+    </div>
+    <%= f.submit "Add Recipe", data: {disable_with: "Adding..."}, class: "btn btn-primary btn-floating mx-1 w-100" %>
+  <% end %>
+</div> 
```

Comment:
- [ ] Create Your Recipe form doesn't work as expected. 
          - If you click on the `Add Recipe` button doesn't send the data 
          - If you check the public recipes you can't see them and it has been clicked by a user when you are filling the form.
          
![recipe](https://user-images.githubusercontent.com/55635977/148658938-ab331486-60e4-4ee9-94b9-fc79fb394c10.png)

Reply:
@Emmyn5600 thank you for reviewing our project but Create Your Recipe is working fine. You did a mistake by putting on "Cooking time(mins)" the incorrect data because it should be a number, not a string.